drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -194,12 +194,13 @@ def referral_requests(**args)
       scoped_referral_requests(object.external_referral_requests, **args)
     end
 
-    # TODO(#186102846) support user-specified sorting/filtering
     def incoming_referral_postings(**args)
       access_denied! unless current_permission?(entity: object, permission: :can_manage_incoming_referrals)
 
+      statuses = HmisExternalApis::AcHmis::ReferralPosting::ACTIVE_STATUSES + [:accepted_status]
+
       scoped_referral_postings(
-        object.external_referral_postings.active, # Only show Active postings on the incoming referral table
+        object.external_referral_postings.where(status: statuses),
```

Comment:
Expanded the scope of the incoming referral table to show `accepted_status` postings as well. This allows providers to see all accepted referrals in one place. They are already able to navigate to them from the Enrollment Dashboard.

The "default" UI is unchanged because the frontend filters out `accepted` by default (https://github.com/greenriver/hmis-frontend/pull/812).